docs/concepts/prefetching.md

```diff
@@ -6,3 +6,92 @@ hide_title: true
 ---
 
 # `Prefetching`
+
+The goal of prefetching is to make data fetch _before_ the user navigates to a page or attempts to load some known content.
+
+There are a handful of situations that you may want to do this, but some very common use cases are:
+
+1. User hovers over a navigation element
+2. User hovers over a list element that is a link
+3. User hovers over a next pagination button
+
+### Prefetching with React Hooks
+
+The `usePrefetch` hook accepts two parameters: the first is the key of a query action, and the second is an object of two optional parameters:
+
+1. `ifOlderThan` - (default: `false` | `number`) - _number is value in seconds_
+
+   - If specified, it will only run the query if the difference between `new Date()` and the last `fulfilledTimeStamp` is greater than the given value
+
+2. `force`
+
+   - If `force: true`, it will ignore the `ifOlderThan` value if it is set and the query will be run even if it exists in the cache.
+
+#### Behavior of the hook
+
+1. If `force: true` is set during the declaration or at the call site, the query will be run no matter what. The one exception to that is if the same query is already in-flight.
+2. If no options are specified and the query exists in the cache, the query will be performed.
+3. If no options are specified and the query _does not exist_ in the cache, the query will be performed.
+   - `isLoading` will be true, `isFetching` will be true
```

Comment:
It's not really clear that this means the value returned from the `useQuery` hook used somewhere else - this could also be interpreted as if the hook would return these flags